modules/ROOT/pages/platform/security/single-sign-on.adoc

```diff
@@ -6,25 +6,100 @@ label:AuraDB-Virtual-Dedicated-Cloud[]
+== Log-in methods
+
+Organization owners and organization admins can configure a combination of one or more of the log-in methods.
+
+*1. Use as a login method for the organization*
+
+* Email/password
+* Okta
+* Microsoft Entra ID
+* Google SSO (not Google Workspace SSO) 
+
+For *Use as a login method for the organization*, organization admins can disable email/password and Google SSO if at least one other custom SSO provider is configured.
+
+*2. Use as a log-in method for instances within Projects in this organization*
+
+* User/password
+* Okta
+* Microsoft Entra ID
+
+Enabling *Use as a log-in method for instances within Projects in this organization* locks the user/password method and prevents it from being disabled.
+
+== Setup requirements
+
+Accessing Aura with SSO requires:
+
+* Authorization Code Flow
+* A publicly accessible IdP server
+
+To create an SSO Configuration either a Discovery URI or a combination of Issuer, Authorization Endpoint, Token Endpoint and JWKS URI is required.
+
+== Create a new SSO configuration
+
+From the *Organization settings*, go to *Single Sign-On* and use the *SSO Configuration* button to set up a new SSO configuration.
+
+.Organization settings
+[.shadow]
+image::organizationsettings.png[A screenshot of how to navigate to organization settings in the UI]
+
+The checkboxes *Use as a log in for the Organization* and *Use as login method for instances with projects in this Org* define whether SSO should be only on Organization level, only on Project level, or both.
+
+The required basic SSO configuration information can be retrieved from the IdP.
+Entering the Discovery URI pre-fills the fields below. 
+If this is not known these fields can be completed manually.
+
+.SSO configuration
+[.shadow]
+image::sso.png[A screenshot of the SSO configuration dialogue,640,480]
+
+.SSO toggles
+[.shadow]
+image::ssotoggles.png[A screenshot of the SSO toggles (for password/user and Google social log in) after SSO configuration dialogue,640,480]
+
+== Individual instance level SSO configurations available from Support
+
+Support can assist with:
+
+* Role mapping specific to a database instance
+* Custom groups claim besides `groups`
+* Updating SSO on already running instances
+
+If you require support assistance, visit link:https://support.neo4j.com/[Customer Support] and raise a support ticket including the following information:
+
+
+. The _Project ID_ of the projects you want to use SSO for.
+See xref:platform/user-management.adoc#_projects[Projects] for more information on how to find your __Project ID__.
+
+. The name of your IdP
+
+== Authorization vs. Authentication
+
+*Authentication* determines who is allowed in. 
+Authentication is handled by Single Sign-On (SSO), which serves as a method for logging in. 
+
+*Authorization* determines what someone can do when they have logged-in.
```

Comment:
No need to explain this to the user, I think we can safely assume that they know the difference if they attempt to set up SSO.

Reply:
I think it’s really useful to include this somewhere, but perhaps not in this part of the docs.

When I was new to RBAC and SSO, Linda Haviv’s explanation helped me a lot—it clearly drew a dividing line between the two concepts for me. Before this, I used to see RBAC and SSO as one merged concept. Her castle metaphor is especially effective:

https://www.linkedin.com/posts/lindahaviv_lets-learn-about-authentication-vs-authorization-activity-7221535846015868929-ssbq